scripts/kokkos_cuda_openmp_cmake

```diff
@@ -51,11 +55,14 @@ cmake \
     -D Trilinos_ENABLE_DataTransferKitLibmeshAdapters:BOOL=OFF \
     -D Trilinos_ENABLE_DataTransferKitClassicDTKAdapters:BOOL=OFF \
     -D Trilinos_ENABLE_OpenMP:BOOL=ON \
+    -D Trilinos_ENABLE_Tpetra:BOOL=ON \
     -D Kokkos_ENABLE_Cuda:BOOL=ON \
     -D Kokkos_ENABLE_Cuda_UVM:BOOL=ON \
     -D Tpetra_INST_SERIAL:BOOL=ON \
     -D Tpetra_INST_OPENMP:BOOL=ON \
     -D Tpetra_INST_CUDA:BOOL=ON \
+    -D Tpetra_INST_COMPLEX_DOUBLE:BOOL=OFF \
+    -D Tpetra_INST_COMPLEX_FLOAT:BOOL=OFF \
```

Comment:
Use a more visually appealing CMake config style:
1. Use bash arrays (ARGS) to allow for line commenting
2. Put package options to it with an extra offset:
```
-D Trilinos_ENABLE_Tpetra:BOOL=ON \
    -D Tpetra_INST_COMPLEX_DOUBLE:BOOL=OFF \
    -D Tpetra_INST_COMPLEX_FLOAT:BOOL=OFF \
```